### PR TITLE
Update to newest version of phantomas

### PR DESCRIPTION
**Pull request by gfdickinson**

### CODE REVIEW

File: tasks/lib/phantomas.js [hunk 1]
```diff
@@ -506,7 +506,7 @@ Phantomas.prototype.getImages = function() {
 
   return _.sortBy( files, _.bind( function( file ) {
     return +file.match(
-      /^screenshot-\d\d\d\d-\d\d-\d\dT\d\d:\d\d:\d\d-(\d*).png$/
+      /^screenshot-\d\d\d\d-\d\d-\d\dT\d\d-\d\d-\d\d-(\d*).png$/
```
Comment (macbre) [inline tasks/lib/phantomas.js]: @stefanjudis, please note that phantomas triggers `filmStrip` event on each screenshot save. May be a better approach then trying to match these files.

Comment (stefanjudis) [inline tasks/lib/phantomas.js]: @macbre thanks for point this out.

This actually only for sorting the files properly. Every run has it's own directory as far as I remember, so there is no problem with it atm. :bowtie: 

Or do I miss the point?

**Comment by stefanjudis** [comment]
@gfdickinson Hmm, fail?

**Comment by gfdickinson** [comment]
@stefanjudis feel free to fail.  I will try to figure out a more elegant solution.

File: tasks/lib/phantomas.js (hunk 1 above)
Comment (macbre) [inline tasks/lib/phantomas.js]: If it's for sorting only, why there is matching?

**Comment by stefanjudis** [comment]
@gfdickinson 

For the tests you probably only have to adjust 

https://github.com/stefanjudis/grunt-phantomas/blob/master/test/lib/phantomasTest.js#L599.

Or do you wanna refactor the whole thing?

**Comment by gfdickinson** [comment]
I will submit a new pull request that fixes the tests for now.

File: tasks/lib/phantomas.js (hunk 1 above)
Comment (stefanjudis) [inline tasks/lib/phantomas.js]: It generated images like:

- `screenshot-2014-11-30T14:06:31-41.png`
- `screenshot-2014-11-30T14:06:31-767.png`
- `screenshot-2014-11-30T14:06:31-1124.png`

I'm grabing everything behind the dash ( which was the time passed by, or ) and use this for sorting.

```
  return _.sortBy( files, _.bind( function( file ) {
    return +file.match(
      /^screenshot-\d\d\d\d-\d\d-\d\dT\d\d:\d\d:\d\d-(\d*).png$/
    )[ 1 ];
  }, this ) );
```

The binding is not needed - but I think it makes sense. :)

**Comment by stefanjudis** [comment]
Whatever you prefer.

Thanks for your effort. :)

**Pull request closed without merging**